src/Model/Context.php

```diff
@@ -98,11 +117,11 @@ public function has($name)
     }
 
     /**
-     * @return array
+     * @return array|\stdClass
      */
     public function jsonSerialize()
     {
-        return $this->data;
+        return $this->data ?: new \stdClass;
```

Comment:
I'm just returning data now. It would't be good to just return null. I can live with the empty array and taking care of it in the ConversationApi.